src/plugins/PluginAPI.cpp

```diff
@@ -194,7 +194,12 @@ APICALL bool HyprlandAPI::addDispatcher(HANDLE handle, const std::string& name,
 
     PLUGIN->registeredDispatchers.push_back(name);
 
-    g_pKeybindManager->m_mDispatchers[name] = handler;
+    std::function<SDispatchResult(std::string)> newHandler = [handler](std::string arg1) -> SDispatchResult {
+        handler(arg1);
+        return {};
+    };
+
+    g_pKeybindManager->m_mDispatchers[name] = newHandler;
```

Comment:
why is this split like that?

Reply:
idk thats how I typed it

forgot to ask but shouldn't we just change api?
or maybe add something like `addDispatcher2`

Reply:
no

Reply:
oke